src/schoology-extractor/tests/helpers/test_sync.py

```diff
@@ -169,19 +165,20 @@ def test_return_current_date_if_there_is_no_record(self):
             result = sync._get_last_modified_date(
                 "fake_table_name",
                 mock_db_engine,
-                "fake_column_id",
                 3)
 
             assert str(result) == "2020-10-16 13:00:01"
 
 
 @pytest.fixture
 def db_engine_mock_returns_fake_resource():
-    # [(key0, value0), (key1, value1)]
     row = Mock()
     row.items.return_value = [
-        ('id', 1),
-        ('name', 'fake_name')
+        ('JsonResponse',
+            r"""{
+                "id": 1,
+                "name": "fake_name"
+            }""")
```

Comment:
`JsonResponse` is only one of the returned keys, for the scope of this test, this key is the only required one.